holopy/scattering/scatterer/composite.py

```diff
@@ -74,45 +85,71 @@ def get_component_list(self):
                 components.append(s)
         return components
 
-    @property
-    def parameters(self):
-        pars = []
-        names = []
-        for i, scatterer in enumerate(self.scatterers):
-            for key, par in scatterer.parameters.items():
-                for index, par_check in enumerate(pars + [None]):
+    def update_ties(self):
```

Comment:
As a general comment / question:

This seems like some pretty powerful stuff here. Is there a reason it's limited to ```Scatterers``` and not to the more general ```Scatterer``` class? For instance, I could imagine wanting to fit an ellipsoid with the two minor axes equal.

If this is not a very easy fix then let's not do it for this PR and do it for a later update when desired.

Reply:
It's not a trivially easy change, but it wouldn't be too hard to do. I just can't think of an obvious use case. In your example, you should just use the `spheroid` class instead of `ellipsoid`.

Reply:
Let's just leave it as is then for now.